cmake/Modules/winconf-msvc.cmake

```diff
@@ -1,21 +1,12 @@
 if (STACKTRACE)
-    if ("${WINXXBITS}" STREQUAL "Win64")
-        add_compile_options(-Zi)
-    else ("${WINXXBITS}" STREQUAL "Win64")
+    if (NOT "${WINXXBITS}" STREQUAL "Win64")
         # i686 arch requires frame pointer preservation
         add_compile_options(-Oy-)
     endif ("${WINXXBITS}" STREQUAL "Win64")
```

Comment:
@glassez:  In case you missed it, the `endif` no longer matches the `if`, here.

Reply:
Thank you! Fixed.